### DISCUSSION TITLE
Profile text needed.

### DISCUSSION THREAD

**Discussion by lg-nineoneninethreenine** [TimerX-App/.github#2] [Announcements]
We need profile text. @Futura-Py/reviewers

**Comment by sumeshir26** [comment]
I think we should copy-paste the TimerX README until we make some other apps which is not anytime soon

**Reply by lg-nineoneninethreenine** [reply]
Hmm... Fine...

**Comment by not-nef** [comment]
Can someone review the profile text i made? @sumeshir26 @im-coder-lg

**Comment by lg-nineoneninethreenine** [comment]
Great!!!
